xla/service/gpu/backend_configs.proto

```diff
@@ -161,4 +161,7 @@ message CudnnfMHABackendConfig {
 
   // Random seed used by dropout
   int64 seed = 15;
+
+  // Is flash attention
+  bool is_flash_attention = 20;
```

Comment:
Is this parameter composable with all the others? E.g. can you have a valid flash attention with any dot dimension numbers/dropout rate/fmha scale/seed/etc?

Reply:
flash attention is valid with any dot dumensions/dropout rate/fmha scale/seed. The constraint is that head dim should be divisible by 8 and up to 128, sequence length Q and K should be equal.